docs/cluster_spec.md

```diff
@@ -384,6 +384,17 @@ spec:
     disableSecurityGroupIngress: true
 ```
 
+#### WARNING: this works only for Kubernetes version above 1.7.0.
+
+For avoid to create security group per each elb, you can specify security group id, that will be assigned to your LoadBalancer. It must be security group id, not name. Also, security group must be empty, because Kubernetes will add rules per ports that are specified in service file.
```

Comment:
Mind fixing up the wording at the beginning while you're here? `For avoid to create...`